src/Products/Viewer/Controllers/ViewerApiController.cs

```diff
@@ -1,47 +1,53 @@
-﻿using GroupDocs.Viewer.MVC.Products.Common.Entity.Web;
-using GroupDocs.Viewer.MVC.Products.Common.Resources;
-using GroupDocs.Viewer.MVC.Products.Viewer.Cache;
-using GroupDocs.Viewer.Caching;
-using GroupDocs.Viewer.Options;
-using System;
-using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.IO;
-using System.Net;
-using System.Net.Http;
-using System.Web.Http;
-using System.Web.Http.Cors;
-using GroupDocs.Viewer.MVC.Products.Viewer.Config;
-using GroupDocs.Viewer.MVC.Products.Common.Util.Comparator;
-using GroupDocs.Viewer.Results;
-using System.Text;
-using GroupDocs.Viewer.Exceptions;
-using GroupDocs.Viewer.Interfaces;
-using System.Xml.Linq;
-using System.Linq;
-using System.Globalization;
-using System.Web;
-using System.Net.Http.Headers;
+﻿// <copyright file="ViewerApiController.cs" company="GroupDocs">
+// Copyright (c) GroupDocs. All rights reserved.
+// </copyright>
 
 namespace GroupDocs.Viewer.MVC.Products.Viewer.Controllers
 {
+    using System;
+    using System.Collections.Concurrent;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.IO;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Text;
+    using System.Web;
+    using System.Web.Http;
+    using System.Web.Http.Cors;
+    using System.Xml.Linq;
+    using GroupDocs.Viewer.Caching;
+    using GroupDocs.Viewer.Exceptions;
+    using GroupDocs.Viewer.Interfaces;
+    using GroupDocs.Viewer.MVC.Products.Common.Entity.Web;
+    using GroupDocs.Viewer.MVC.Products.Common.Resources;
+    using GroupDocs.Viewer.MVC.Products.Common.Util.Comparator;
+    using GroupDocs.Viewer.MVC.Products.Viewer.Cache;
+    using GroupDocs.Viewer.MVC.Products.Viewer.Config;
+    using GroupDocs.Viewer.Options;
+    using GroupDocs.Viewer.Results;
+
     /// <summary>
-    /// ViewerApiController
+    /// ViewerApiController.
     /// </summary>
     [EnableCors(origins: "*", headers: "*", methods: "*")]
     public class ViewerApiController : ApiController
     {
-        private static Common.Config.GlobalConfiguration globalConfiguration;
+        /// <summary>
+        /// Map for locking keys in viewer cache.
+        /// </summary>
+        public static readonly ConcurrentDictionary<string, object> KeyLockerMap = new ConcurrentDictionary<string, object>();
```

Comment:
Codacy found an issue: [Use an immutable collection or reduce the accessibility of the field(s) 'KeyLockerMap'.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=5850423)